plugins/legacy-plugin-chart-sankey/src/Sankey.js

```diff
@@ -187,7 +204,7 @@ function Sankey(element, props) {
     .attr('dy', '.35em')
     .attr('text-anchor', 'end')
     .attr('transform', null)
-    .text(d => d.name)
+    .text(d => (responsiveClass === 'xs' ? '' : d.name))
```

Comment:
I wonder if we should have a class name for hiding the labels other than `xs` (no good ideas to propose right now, but thinking out loud)